openlibrary/plugins/worksearch/code.py

```diff
@@ -871,6 +873,29 @@ def GET(self):
         return delegate.RawText(json.dumps(response))
 
 
+@public
+def random_author_search(limit=10):
+    """
+    Returns a JSON string that contains a random list of authors.  Amount of authors
+    returned is set be the given limit.
+    """
+    s = string.ascii_letters + string.digits
```

Comment:
```suggestion
    letters_and_digits = string.ascii_letters + string.digits
```
Can we please try to avoid single letter variable names?

Reply:
Thanks!  I got a bit hung up on what to call that...